packages/components/src/internal/actions.ts

```diff
@@ -770,7 +763,7 @@ export function fetchCharts(schemaQuery: SchemaQuery, containerPath?: string): P
                     resolve(result);
                 } else {
                     reject({
-                        error: 'study-report-getReportInfos.api responded to success without success',
+                        error: 'report-getReportInfos.api responded to success without success',
```

Comment:
Hmm. Perhaps we could improve this messaging....

Reply:
haha, but it is so helpful...
I'll update. Thanks